### ISSUE TITLE
Slide preview images still fuzzy

### ISSUE THREAD

**Issue by oas777** [elan-ev/tobira#1230]
Mentioned in https://github.com/elan-ev/tobira/issues/1196, this was not solved yet, says https://tobira.opencast.org/!v/JePe-XwDwdz.

**Comment by owi92** [comment]
Yep right, though that needs to be adjusted in Opencast if I understood @LukasKalbertodt correctly.

**Comment by oas777** [comment]
@wsmirnow, is Opencast responsible for the resolution of the slide preview images in Paella?

**Comment by wsmirnow** [comment]
Yes, it is.

**Comment by oas777** [comment]
Reopened because of https://tobira.opencast.org/!v/Fz-M4ET6d7F:

![Image](https://github.com/user-attachments/assets/0781d299-7581-435d-97bc-3fff485ec811)

**Comment by LukasKalbertodt** [comment]
We briefly talked about this just now. But to repeat: the problem here is Opecast, which generates low-res images. It is fixed for your ETH system, but not for the Opencast behind tobira.opencast.org. The default resolution should be adjusted in community workflows, such that when we next update "our" Opencast, that we also get higher resolution images.

But yes, I don't think this issue needs to be open here, as it has nothing to do with Tobira.

**Comment by oas777** [comment]
This should be of concern to everyone who is about to launch an OC Explore to showcase our product ... is https://github.com/opencast/opencast/issues the place to add the issue to the existing 223?

**Comment by LukasKalbertodt** [comment]
> This should be of concern to everyone who is about to launch an OC Explore to showcase our product ... is https://github.com/opencast/opencast/issues the place to add the issue to the existing 223?

Yep, this really should get fixed in the community. And yes, that is the right place to open an issue about this. I am just not sure what you mean by 223.

**Comment by oas777** [comment]
Cf. https://github.com/opencast/opencast/issues/6536. That makes it 225 open issues, not 223. Is https://github.com/opencast/opencast/issues/3753 the more general problem?
